pipelines/services/types.go

```diff
@@ -0,0 +1,32 @@
+package services
+
+import "time"
+
+type PipResponse struct {
```

Comment:
Pip is occupied by Python.
Also, this struct represents the RunStatus response.
```suggestion
type PipelinesRunStatus struct {
```